src/components/layout/TopBar/TopBar.module.scss

```diff
@@ -1,5 +1,13 @@
 @import "../../../styles/settings.scss";
 
+$tablet-width: 768px;
+
+@mixin mobile {
```

Comment:
To przenieśmy do jednego wspólnego pliku

Reply:
Przeniesione